videos/templates/mail/youtube_upload_failure/body.html

```diff
@@ -0,0 +1,22 @@
+{% extends "mail/email_base.html" %}
+
+{% block content %}
+<tr>
+  <td style="padding: 20px;">
+    <table role="presentation" cellspacing="0" cellpadding="0" border="0" width="100%">
+      <tr>
+        <td style="padding: 20px; font-family: sans-serif; font-size: 15px; line-height: 20px; color: #555555;">
+          <p style="margin: 0 0 10px;">Dear {{ user.name }},</p>
+          <p style="margin: 0 0 10px;">
+            The video "{{ video.filename }}" for your site <a href="{{ site.url }}">{{ site.title }}</a>
+            could not be uploaded to Youtube. Please contact OCW support for help. Error:
+          </p>
+          <p>
+              {{ video.error }}
```

Comment:
Do we want to have the error here? Do you think it would be useful for people outside our team. The error I got while testing from a misconfigured call to the webhook
```
unable to access bucket: 'ol-ocw-studio-app-qa' key: 'aws_mediaconvert_transcodes/anastasia_test_site_4/1v0oGQfMEdyjXfDOW9AWEWU3Yu5YIDKti/jellie6s_360p_16_9.mp4/jellie6s_youtube.mp4' version: None error: An error occurred (NoSuchKey) when calling the GetObject operation: The specified key does not exist.
```
Seems not very useful to anyone but engineers but maybe the errors encountered by actual users will be more actionable?

Reply:
Makes sense, I removed it from the message body.  We should be able to figure it out from sentry.